src/sage/plot/matrix_plot.py

```diff
@@ -224,10 +224,10 @@ def _render_on_subplot(self, subplot):
             extent = (lim['xmin'], lim['xmax'],
                       lim['ymax' if flip_y else 'ymin'],
                       lim['ymin' if flip_y else 'ymax'])
-            opts = dict(cmap=cmap, interpolation='nearest', aspect='equal',
-                      norm=norm, vmin=options['vmin'], vmax=options['vmax'],
-                      origin=('upper' if flip_y else 'lower'),
-                      extent=extent, zorder=options.get('zorder'))
+            opts = {'cmap': cmap, 'interpolation': 'nearest', 'aspect': 'equal',
```

Comment:
Do we consider this style change an improvement and why?

Reply:
well, this avoids using the `dict` word. Not clear to me if this is really an improvement. I can certainly undo changes in this file.

Reply:
Is this suggestion coming from ruff? Does it have explanations/references for its style codes?

Reply:
This is ruff C408 : https://docs.astral.sh/ruff/rules/unnecessary-collection-call/

having to call dict is slower, they say

Reply:
I don't know. Either way is fine with me. But if you want this change, please adjust the indentation